app/.coveragerc

```diff
@@ -2,6 +2,8 @@
 [run]
 source = experimenter
 #branch = True
+omit =
+    */tests/*
```

Comment:
This is not required for this commit but I noticed coverage was including the tests files themselves, which is weird.